### PR TITLE
Dmceachernmsft/MeetingCompositeLocaleSidePane

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What
Add locale string use to the side pane in the meeting composite

# Why
Complete locale integration into the meeting composite.

# How Tested
Ran Locally with locale strings added in the storybook sample.

### CODE REVIEW

**Review by alkwa-msft — APPROVED** [review]

**Review by mgamis-msft — APPROVED** [review]

File: packages/react-components/src/localization/locales/en-GB/strings.json [hunk 1]
```diff
@@ -15,12 +15,17 @@
...
-    "textTooLong": "Your message length is over the maximum limit."
+    "textTooLong": "Your message length is over the maximum limit.",
+    "sendButtonAriaLabel": "Send message"
   },
   "messageStatusIndicator": {
+    "deliveredAriaLabel": "Message sent",
     "deliveredTooltipText": "Sent",
+    "seenAriaLabel": "Message seen by others",
     "seenTooltipText": "Seen",
+    "sendingAriaLabel": "Message sending",
```
Comment (prprabhu-ms) [inline packages/react-components/src/localization/locales/en-GB/strings.json:26]: Should this be "Sending message" instead?

Actually, "Sent message", "Failed to send message" might also sound better when announced by the narrator.

File: packages/react-components/src/localization/locales/de-DE/strings.json [hunk 2]
```diff
@@ -15,12 +15,17 @@
   },
   "sendBox": {
     "placeholderText": "Nachricht eingeben",
-    "textTooLong": "Ihre Nachrichtenlänge überschreitet den maximalen Grenzwert."
+    "textTooLong": "Ihre Nachrichtenlänge überschreitet den maximalen Grenzwert.",
+    "sendButtonAriaLabel": "Nachricht senden"
```
Comment (prprabhu-ms) [inline packages/react-components/src/localization/locales/de-DE/strings.json:19]: You don't need to update any of the translated strings. Just the en-US ones. The rest will be updated via autmation / td build.

File: packages/react-composites/src/composites/localization/locales/es-ES/strings.json [hunk 3]
```diff
@@ -41,5 +41,12 @@
   },
   "chat": {
     "chatListHeader": "En este chat"
+  },
+  "meeting": {
+    "peopleButtonLabel": "Contactos",
```
Comment (prprabhu-ms) [inline packages/react-composites/src/composites/localization/locales/es-ES/strings.json:46]: Typo?

File: packages/react-composites/src/composites/localization/locales/es-ES/strings.json [hunk 4]
```diff
@@ -41,5 +41,12 @@
   },
   "chat": {
     "chatListHeader": "En este chat"
+  },
+  "meeting": {
+    "peopleButtonLabel": "Contactos",
+    "chatButtonLabel": "Chat",
+    "peoplePaneTitle": "Contactos",
```
Comment (prprabhu-ms) [inline packages/react-composites/src/composites/localization/locales/es-ES/strings.json:48]: Typo?

File: packages/react-composites/src/composites/localization/locales/es-ES/strings.json (hunk 3 above)
Comment (prprabhu-ms) [reply packages/react-composites/src/composites/localization/locales/es-ES/strings.json:46]: nvm, this is not English :P

File: packages/react-composites/src/composites/localization/locales/es-ES/strings.json (hunk 4 above)
Comment (prprabhu-ms) [reply packages/react-composites/src/composites/localization/locales/es-ES/strings.json:48]: nvm, this is not English :P

**Review by prprabhu-ms — APPROVED** [review]

File: packages/react-components/src/localization/locales/de-DE/strings.json (hunk 2 above)
Comment (dmceachernmsft) [reply packages/react-components/src/localization/locales/de-DE/strings.json:19]: I just ran the td build to update the strings to make sure that the logic I wrote was correct.

**Pull request merged**